config/standalone.dev.webpack.config.js

```diff
@@ -0,0 +1,37 @@
+const webpackBase = require('./webpack.base.config');
+
+// Used for getting the correct host when running in a container
+const proxyHost = process.env.API_PROXY_HOST || 'localhost';
+const proxyPort = process.env.API_PROXY_PORT || '5001';
+const apiBasePath = process.env.API_BASE_PATH || '/api/ansible-catalog/v1';
```

Comment:
Are we going to need to change these to something less generic? For example for the API, all of the environment variables are going to be prefixed with `ANSIBLE_CATALOG_`, should the same be true for these? We probably need to change the ones in catalog-ui, too.

Reply:
@eclarizio - agree - we should change the names to use the same prefix